package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "@divriots/dockit-core",
-  "version": "0.1.8",
+  "version": "0.1.9",
   "description": "",
   "license": "ISC",
   "type": "module",
   "homepage": "{{ homepage }}",
   "scripts": {
-    "serve": "dsd-cli serve 4BeMe20hqOWTkdUL2NuJ --open"
+    "serve": "npx backlight@latest serve 4BeMe20hqOWTkdUL2NuJ --open"
+  },
+  "dependencies": {
+    "copy-to-clipboard": "^3.3.1",
+    "lodash": "^4.17.21",
+    "tailwindcss": "^2.2.15"
   },
   "devDependencies": {
-    "dsd-cli": "latest"
+    "lit": "^2.0.0-rc.4",
```

Comment:
Not sure, but shouldn't twind and lit be standard deps of dockit-core?
Technically speaking twind should be a runtime dependency of dockit-core

Reply:
Nope, they're both only used inside the stories:
- lit as a renderer to render the story
- twind to import the twind shim for the dockit-tailwind-showcases demo, but the shim is not a dependency of the component itself. Or, if it is, then this import should be moved into the src of the component, maybe you can explain why it is or why it isn't a runtime dependency of the component?

Reply:
Let's wait on Georges' feedback on your other PR, it's not clear to me either
My point was just that **twind** is used to compute the tailwind styles for the doc, and it does have to be setup/called somewhere to work. Sounded to me like the best place was the showcases component themselves